packages/relay/src/lib/eth.ts

```diff
@@ -526,8 +528,11 @@ export class EthImpl implements Eth {
       return null;
     }
 
+    const maxPriorityFee = contractResult.max_priority_fee_per_gas === EthImpl.emptyHex ? undefined : contractResult.max_priority_fee_per_gas;
+    const maxFee = contractResult.max_fee_per_gas === EthImpl.emptyHex ? undefined : contractResult.max_fee_per_gas;
+    
     return new Transaction({
-      accessList: contractResult.access_list,
+      accessList: undefined, // we don't support access lists, so punt for now
```

Comment:
Q: Why set `undefined` vs `null`?

Reply:
Schema doesn't support null values.  Undefined keeps it out of the serialized form and makes it clear we need to fix it when we support access  lists.

Reply:
Okay, wanted to make sure the omission was intended behaviour.